plugins/nf-google/src/test/nextflow/cloud/google/lifesciences/GoogleLifeSciencesTaskHandlerTest.groovy

```diff
@@ -197,7 +197,7 @@ class GoogleLifeSciencesTaskHandlerTest extends GoogleSpecification {
         then:
         task.getConfig() >> new TaskConfig(containerOptions: [entrypoint:null])
         and:
-        req.entryPoint == null
+        req.entryPoint == GoogleLifeSciencesConfig.DEFAULT_ENTRY_POINT
 
```

Comment:
Interesting, why before it was null? 😕

Reply:
So, the constant is `/bin/bash`

https://github.com/nextflow-io/nextflow/blob/96c3fd4e79ddfd028380fc0b01aef3494450c59e/plugins/nf-google/src/main/nextflow/cloud/google/lifesciences/GoogleLifeSciencesConfig.groovy#L48-L48

However when is null defaults to `bash` 

https://github.com/nextflow-io/nextflow/blob/96c3fd4e79ddfd028380fc0b01aef3494450c59e/plugins/nf-google/src/main/nextflow/cloud/google/lifesciences/GoogleLifeSciencesHelper.groovy#L245-L246

Think, it would be better to stick to the current behaviour to avoid introducing unexpected side effects. I'll do the chanege

Reply:
Even more intriguing.. actually when `entrypoint` was not specified it was defaulting to `/bin/bash` 

https://github.com/nextflow-io/nextflow/blob/96c3fd4e79ddfd028380fc0b01aef3494450c59e/plugins/nf-google/src/test/nextflow/cloud/google/lifesciences/GoogleLifeSciencesTaskHandlerTest.groovy#L185-L185

Not clear why there was that test checking for `null`

Reply:
I've removed that tests 😆